web/routes.go

```diff
@@ -63,3 +70,17 @@ func initiateTranscriptionJobHandler(w http.ResponseWriter, r *http.Request) {
 func healthHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("healthy!"))
 }
+
+var statuses = [...]string{
+	"In progress",
+	"Done",
+	"Error",
+}
+
+func jobStatusHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
@rachelhongca can you add a doc comment (for reference on Go comments, you can refer to: https://golang.org/doc/effective_go.html#commentary)